cleancode/src/main/java/com/epam/cleancode/comments/MortgageInstallmentCalculator.java

```diff
@@ -1,39 +1,36 @@
 package com.epam.cleancode.comments;
 
-public class MortgageInstallmentCalculator {
+class MortgageInstallmentCalculator {
 
     /**
      *
-     * @param p principal amount
-     * @param t term of mortgage in years
-     * @param r rate of interest
+     * @param LoanAmount principal amount
+     * @param TermDurationInYears term of mortgage in years
+     * @param RateOfInterest rate of interest
      * @return monthly payment amount
      */
-    public static double calculateMonthlyPayment(
-            int p, int t, double r) {
+    static double calculateMonthlyPayment(int LoanAmount, int TermDurationInYears, double RateOfInterest) {
 
         //cannot have negative loanAmount, term duration and rate of interest
-        if (p < 0 || t <= 0 || r < 0) {
+        if (LoanAmount < 0 || TermDurationInYears <= 0 || RateOfInterest < 0) {
             throw new InvalidInputException("Negative values are not allowed");
```

Comment:
if you will show it to user better to move it into resources with possibilities of translations